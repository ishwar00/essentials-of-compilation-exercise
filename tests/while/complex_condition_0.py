i = 0
j = 9
while i < 9 or j > 0:
    print(i)
    print(j)
    i = i + 1
    j = j - 1
