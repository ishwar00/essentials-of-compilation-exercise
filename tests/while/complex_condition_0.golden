091827364554637281
